dactylo.html

```diff
@@ -24,15 +24,15 @@ <h1 class="key_list">·</h1>
     <p style="text-align: right;">
       <span style="float: left;">
         <select id="layout">
-          <option selected>ergol</option>
-          <!-- <option>erglace</option> -->
-          <option>bepo</option>
-          <option>azerty</option>
-          <option>lafayette</option>
+          <option value="ergol" selected>Ergo‑L</option>
+          <!-- TODO: Erglace -->
+          <option value="bepo">Bépo</option>
+          <option value@"azerty">AZERTY</option>
```

Comment:
Ntpicks / questions : Why is `AZERTY` in full caps ? Is the `@` in `value@"` a typo ?

Reply:
AZERTY is in full caps because that’s its real name. The `@` is totally a typo.

Reply:
I agree AZERTY looks horrible in full caps. Reverting.